packages/visualizations-react/stories/Table/Table.stories.tsx

```diff
@@ -76,4 +76,10 @@ export const Unstyled = Template.bind({});
 Unstyled.args = {
     data,
     options: { ...options, unstyled: true },
+};
+
+export const emptyState = Template.bind({});
+emptyState.args = {
+    data: { value: [], loading: false },
+    options: { ...options, emptyStateLabel: 'Neniuj registroj trovitaj...' },
```

Comment:
Story should show 'Neniuj registroj trovitaj...'

Prop drilling to `Body.svelte` is missing